### ISSUE TITLE
Interpolation doesn't work with new line characters (i.e. \n and \r)

### ISSUE THREAD

**Issue by DzmitryShylovich** [angular/angular#13465]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

Input:
```
{{'foo' +
    'bar'}}
```
Output:
`{{'foo' + 'bar'}}`

**Minimal reproduction of the problem with instructions**
https://plnkr.co/edit/xCmAnQMQxTseorxtr1c4?p=preview

Angular version: 2.3.0

Looks like regression https://github.com/angular/angular/pull/6056

**Comment by vicb** [comment]
Oh yep it remember that we had already fixed that once :(

**Comment by vicb** [comment]
But what is not working ?

**Comment by ericmartinezr** [comment]
@vicb it prints

```
foobar
```
when it should print
```
foo
bar
```

The same happens with `\r\n` (try it in the plnkr)

```
`{{'foo\r\nbar'}}

// prints

foo bar

// it should print
foo
bar
```
A user came up with this [issue on gitter](https://gitter.im/angular/angular?at=5851880baf6b364a29b15e87), or are we misunderstanding something?

**Comment by pkozlowski-opensource** [comment]
Try this is your favourite HTML editor and a browser:

```html
  <p>foo
  bar</p>
```

**Comment by pkozlowski-opensource** [comment]
tl;dr; is that if you want to have a new line _displayed in the resulting page_ you need to use `<br>` and not `\n`... Unless I'm misunderstanding your problem / question....
